debian/libmiral6.symbols

```diff
@@ -424,3 +424,4 @@ libmiral.so.6 libmiral6 #MINVER#
  MIRAL_4.1@MIRAL_4.1 4.1.0
  (c++)"miral::WaylandExtensions::zwp_input_method_v1@MIRAL_4.1" 4.1.0
  (c++)"miral::WaylandExtensions::zwp_input_panel_v1@MIRAL_4.1" 4.1.0
+ (c++)"miral::X11Support::X11Support(bool)@MIRAL_4.2" 4.1.0
```

Comment:
We need a MIRAL_4.2 for any new entry point.

Reply:
Bumping `MIRAL_VERSION_MINOR` and running `check-and-update-debian-symbols.py` will do that